examples/server_hardware.py

```diff
@@ -82,6 +82,20 @@
 print("Added rack mount server '%s'.\n  uri = '%s'" %
       (server['name'], server['uri']))
 
+# Create Multiple rack-mount servers
+if oneview_client.api_version >= 600:
+    options_to_add_mult = {
+        "mpHostsAndRanges": config['server_mpHostsAndRanges'],
+        "username": config['server_username'],
+        "password": config['server_password'],
+        "licensingIntent": "OneView",
+        "configurationState": "Managed",
+        "initialScopeUris": ["/rest/scopes/7a7db914-ca1d-4112-9189-4ce1a4774ce3"]
+    }
+    server_mult = oneview_client.server_hardware.add_multiple(options_to_add_mult)
+else:
```

Comment:
This `else` is debatable.

Thought behind adding an `else` here being, this endpoint does not return anything. Doesn't throw any exception as well if a lower API version used.